src/diffpy/srreal/JeongPeakWidth.cpp

```diff
@@ -30,14 +30,16 @@ using namespace std;
 // Constructors --------------------------------------------------------------
 
 JeongPeakWidth::JeongPeakWidth() :
-    mdelta1(0.0), mdelta2(0.0), mqbroad(0.0)
+    mdelta1(0.0), mdelta2(0.0), mqbroad(0.0), mqbroad_new(0.0)
```

Comment:
I think this is not a good name.  A more descriptive name would be better.  I don't know, ``separable_mqbroad`` or something along those lines?